install_config/install/docker_registry.adoc

```diff
@@ -1511,6 +1511,28 @@ configuration file] lacks mandatory entries in the
 xref:docker-registry-configuration-reference-middleware[middleware section].
 Add them, re-deploy the registry, and restart your builds.
 
+[[known-issue-prune-fails-due-to-delete-disabled]]
+==== Image Pruning Fails
+
+If you encounter the following error when pruning images:
+
+----
+BLOB sha256:49638d540b2b62f3b01c388e9d8134c55493b1fa659ed84e97cb59b87a6b8e6c error deleting blob
+----
+
+And your xref:viewing-logs[registry log] contains the following information:
+
+----
+error deleting blob \"sha256:49638d540b2b62f3b01c388e9d8134c55493b1fa659ed84e97cb59b87a6b8e6c\": operation unsupported
+----
+
+It means that your xref:advanced-overriding-the-registry-configuration[custom
+configuration file] lacks mandatory entries in the
```

Comment:
I've put a comment next to that config option with info about it being mandatory for image pruning.